### PR TITLE
Observations: Add indicators in Zones and Plots containing 25m sq plots

### PR DESCRIPTION
**Pull request by constanzauanini**
Also added "Last observation: ... " in schedule observation modal

### CODE REVIEW

File: src/scenes/ObservationsRouter/details/ObservationDetailsRenderer.tsx [hunk 1]
```diff
@@ -14,13 +14,18 @@ const ObservationDetailsRenderer =
   (props: RendererProps<TableRowType>): JSX.Element => {
     const { column, row, value } = props;
 
+    const has25mPlots = row.plantingSubzones
+      ?.flatMap((subzone: { monitoringPlots: any[] }) => subzone.monitoringPlots.flatMap((plot) => plot.sizeMeters))
+      .some((size: number) => size.toString() === '25');
+
```
Comment (nickgraz) [inline src/scenes/ObservationsRouter/details/ObservationDetailsRenderer.tsx:20]: Perhaps this should be a static util function that receives subzones? It seems to be duplicated 3 times.

Comment (constanzauanini) [reply src/scenes/ObservationsRouter/details/ObservationDetailsRenderer.tsx]: It is used twice for subzones, so I added a util for that. The other checks the zone.

File: src/scenes/ObservationsRouter/details/index.tsx [hunk 2]
```diff
@@ -134,6 +134,14 @@ export default function ObservationDetails(props: ObservationDetailsProps): JSX.
     }
   }, [zoneNames, searchProps.filtersProps]);
 
+  const has25mPlots = details?.plantingZones
+    .flatMap((zone) =>
```
Comment (nickgraz) [inline src/scenes/ObservationsRouter/details/index.tsx:138]: Can you use the `has25mPlots` function here?

**Review by nickgraz — APPROVED** [review]
Looks good, one more suggestion for improvement but not absolutely necessary

**Pull request merged**